lumicks/pylake/calibration.py

```diff
@@ -310,12 +360,57 @@ def from_field(hdf5, force_channel, time_field="Stop time (ns)") -> "ForceCalibr
         items = []
         for calibration_item in hdf5["Calibration"].values():
             if force_channel in calibration_item:
-                attrs = calibration_item[force_channel].attrs
+                attrs = dict(calibration_item[force_channel].attrs)
                 if time_field in attrs.keys():
+                    # Copy the timestamp at which the calibration was applied into the item
+                    attrs["Timestamp (ns)"] = calibration_item.attrs.get("Timestamp (ns)")
                     items.append(ForceCalibrationItem(attrs))
 
         return ForceCalibration(time_field=time_field, items=items)
 
+    def print_summary(self, tablefmt):
```

Comment:
how does this differ from the `_print_properties` output from the mixin? I'm super confused at this point

Reply:
This is unrelated to the mixin since this isn't the calibration item. This is the list of calibration items. Maybe renaming it would help?

I think part of what makes this iffy is some of the naming. I feel calling this `ForceCalibration` was a mistake (but that's a name that wasn't introduced in this PR).

I think a lot of the mess could already be cleared up with three changes:
- Change the name of this thing to `ForceCalibrationList`.
- Moving the mixin to `detail`.
- Moving the ForceCalibrationItem to its own file.

I feel a combination of questionable naming (legacy) and having too much in one file is making this seem messier than it needs to be already.

In reality, the specialization classes are actually relatively small.